lib/lockfile.js

```diff
@@ -65,7 +65,13 @@ async function getNewLockfile ({ packageJson, lock, isNpm, repositoryTokens }) {
     })
       .then(result => {
         jobCountByServer[nextServer]--
-        return result
+        if ((result instanceof Error) || result.message) {
+          // result is either `error`, `{ok: false}`, `{ok: false, error}` or `{ok: true, contents}`
```

Comment:
no need to fix this now, but that is clearly a terrible interface that we could have done better and should revisit in the future ;D (I take full responsibility for that :)